test/interval_tests/reduction.jl

```diff
@@ -0,0 +1,59 @@
+using IntervalArithmetic
+using Test
+
+setprecision(Interval, Float64)
+
+@testset "Reduction Test" begin
+    @test mpfr_vector_sum(Base.vect(1.0, 2.0, 3.0), 0.5) == 6.0
```

Comment:
Why don't you just use the normal Julia syntax `[1.0, 2.0, 3.0]` to create the vector?

Reply:
You should make test cases where the result depends on the rounding direction to test that that is working.

Reply:
Are there tests of this in the ITF1788 suite?

Reply:
These tests were in the ITF test but only for the Nearest.